components/nav/Header.vue

```diff
@@ -53,22 +53,15 @@ export default {
       </div>
     </div>
 
-    <div class="apps">
-      <nuxt-link v-if="currentCluster" :to="{name: 'c-cluster-apps', params: { cluster: currentCluster.id }}" class="btn role-tertiary">
-        <i class="icon icon-lg icon-marketplace pr-5" />
-        {{ t('nav.apps') }}
-      </nuxt-link>
-    </div>
-
     <div class="top">
       <NamespaceFilter v-if="clusterReady && currentProduct && currentProduct.showNamespaceFilter" />
       <WorkspaceSwitcher v-else-if="clusterReady && currentProduct && currentProduct.showWorkspaceSwitcher" />
     </div>
 
     <div class="back">
-      <a v-if="currentProduct" class="btn role-tertiary" :href="(currentProduct.inStore === 'management' ? backToRancherGlobalLink : backToRancherLink)">
+      <button v-if="currentProduct" class="btn role-tertiary" :href="(currentProduct.inStore === 'management' ? backToRancherGlobalLink : backToRancherLink)">
```

Comment:
This is intentionally a real link so you can optionally middle-click it to open in a new window.  Also you can't just change an `<a>` to a `<button>`, it's not going to go anywhere anymore (or may reload the whole page because it's implicitly `type="select"`, but won't go where intended).

Reply:
I suspected this was the case. Do you think we should omit the case of `<button>` and only use `<a>`?